app/controllers/stealth/controller/replies.rb

```diff
@@ -86,7 +89,12 @@ def say(reply, thread_id: nil)
 
           formatted_reply = handler.send(reply_instance.reply_type)
 
-          client = service_client.new(reply: formatted_reply, thread_id: thread_id)
+          client = service_client.new(
+            reply: formatted_reply,
+            thread_id: thread_id,
+            delete_message: delete_message
+          )
```

Comment:
Let's make this dynamic so it builds the params based on what is needed for the service_client.

Again, this will make our life easier when we get to bandwidth.